service/client/client.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"github.com/derekparker/delve/service/api"
+)
+
+// Interface represents a debugger service client.
+type Interface interface {
```

Comment:
I don't like naming interfaces `Interface`. How about moving this up into the service package and naming the interface `Client`.